dt-core/global-functions.php

```diff
@@ -422,11 +422,35 @@ function render_field_for_display( $field_key, $fields, $post, $show_extra_contr
                         </option>
                     <?php endforeach; ?>
                 </select>
+            <?php elseif ( $field_type === "tags" ) : ?>
+                <div class="tags">
+                    <var id="<?php echo esc_html( $display_field_id ); ?>-result-container" class="result-container"></var>
+                    <div id="tags_t" name="form-tags" class="scrollable-typeahead typeahead-margin-when-active">
```

Comment:
I think id needs to be the $display_field_id too, so we don't have multiple fields with the same id

Reply:
Ha. That was fast :)